features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/NewCustomizedStepper.jsx

```diff
@@ -0,0 +1,427 @@
+import Alert from 'AppComponents/Shared/Alert';
+import Grid from '@material-ui/core/Grid';
+import StepConnector from '@material-ui/core/StepConnector';
+import ApiContext, { useAPI } from 'AppComponents/Apis/Details/components/ApiContext';
+import CheckIcon from '@material-ui/icons/Check';
+import CloseIcon from '@material-ui/icons/Close';
+import { Link } from 'react-router-dom';
+import grey from '@material-ui/core/colors/grey';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Button from '@material-ui/core/Button';
+import AuthManager from 'AppData/AuthManager';
+import Typography from '@material-ui/core/Typography';
+
+
+const ColorlibConnector = withStyles({
+    alternativeLabel: {
+        top: 22,
+    },
+    active: {
+        '& $line': {
+            backgroundImage:
+                'linear-gradient(to left, #50BCEC 50%, #B1D31E 50%)',
+        },
+    },
+    completed: {
+        '& $line': {
+            backgroundImage:
+                'linear-gradient( #B1D31E, #B1D31E)',
+        },
+    },
+    line: {
+        height: 3,
+        border: 0,
+        backgroundColor: '#eaeaf0',
+        borderRadius: 1,
+    },
+})(StepConnector);
+
+const useColorlibStepIconStyles = makeStyles({
+    root: {
+        backgroundColor: '#ccc',
+        zIndex: 1,
+        color: '#fff',
+        width: 56,
+        height: 56,
+        display: 'flex',
+        borderRadius: '50%',
+        justifyContent: 'center',
+        alignItems: 'center',
+        border: '6px solid #E2E2E2',
+    },
+    active: {
+        backgroundColor: '#50BCEC',
+        border: '6px solid #E2E2E2',
+    },
+    completed: {
+        backgroundColor: '#B1D31E',
+        border: '6px solid #E2E2E2',
+    },
+});
+
+function ColorlibStepIcon(props) {
+    const classes = useColorlibStepIconStyles();
+    const { active, completed } = props;
+
+    return (
+        <div
+            className={clsx(classes.root, {
+                [classes.active]: active,
+                [classes.completed]: completed,
+            })}
+        />
+    );
+}
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        width: '100%',
+    },
+    button: {
+        marginRight: theme.spacing(1),
+    },
+    instructions: {
+        marginTop: theme.spacing(1),
+        marginBottom: theme.spacing(1),
+    },
+    iconTrue: {
+        display: 'block',
+        justifyContent: 'flex-start',
+        alignItems: 'center',
+        backgroundColor: '#B1D31E',
+        zIndex: 1,
+        color: '#fff',
+        width: 15,
+        height: 15,
+        borderRadius: '50%',
+    },
+    iconFalse: {
+        color: '#fff',
+        display: 'block',
+        justifyContent: 'flex-start',
+        alignItems: 'center',
+        backgroundColor: grey[500],
+        zIndex: 1,
+        width: 15,
+        height: 15,
+        borderRadius: '50%',
+    },
+}));
+
+function getSteps() {
+    return ['Develop', 'Deploy', 'Test', 'Publish'];
```

Comment:
Shall we move this to a simple constant ? not a function ?

Reply:
Fixed